hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/hevm/HederaEvmTransaction.java

```diff
@@ -127,4 +127,23 @@ public long offeredGasCost() {
     public boolean requiresFullRelayerAllowance() {
         return offeredGasPrice == 0L;
     }
+
+    /**
+     * @return a copy of this transaction with the given {@code exception}
+     */
+    public HederaEvmTransaction withException(@Nullable final HandleException exception) {
```

Comment:
I suppose if you pass `null` here you can _remove_ an exception, but is that ever going to happen (intentionally)?

Reply:
fixed to be @NullNull